src/game/QuizBuilder.ts

```diff
@@ -3,7 +3,7 @@ import { singleChoiceQuestions } from "../questions/singleChoiceQuestions";
 import { textInputQuestions } from "../questions/textInputQuestions";
 import { Question } from "../questions/types";
 import { Quiz } from "./Quiz";
-import { shuffleOrder } from "./shuffle";
+import { shuffleOrder, randomNumber } from "./extraFeatureForBuildQuiz";
```

Comment:
These methods can be used for anything. By using that name you imply it's not for general purposes and a direct dependency for buildQuiz ..